ldscript.txt

```diff
@@ -138,7 +138,7 @@ SECTIONS
         build/sa2/src/game/trapped_animals.o(.text);
         build/sa2/src/game/game_7.o(.text);
         build/sa2/asm/game_7.o(.text);
-        build/sa2/src/game/stage/results_init.o(.text);
+        build/sa2/src/game/stage/boss_to_results_transition.o(.text);
```

Comment:
Why not boss_results_transition?

Reply:
Being slightly mode explicit doesn't hurt ^^

Reply:
The function could be named something like this, I prefer the module names to be short